lib/lotus/model/adapters/sql/console.rb

```diff
@@ -0,0 +1,35 @@
+require 'lotus/model/adapters/sql/sqlite_console'
+require 'lotus/model/adapters/sql/psql_console'
+require 'lotus/model/adapters/sql/mysql_console'
```

Comment:
what do you think about loading on-demand in `#console` method instead loading all?